capstone/src/main/webapp/script.js

```diff
@@ -147,7 +147,7 @@ function addSubstring(substring, analysisContainer, result, loadingEl, response,
   // Break up and color the segment appropriately	
   const substringEl = document.createElement('span');
   const wordElts = [];
-  const words = getSubstrings(substring, TokenizerEnum['WORD']);
+  const words = getSubstrings(substring, /(\S+\s*)/g);
```

Comment:
These are actually not supposed to work 100% the same because of the way I fully preserve the format of the input (newlines, spaces, characters between "segments"). I updated the code with the correct regular expressions, but I do realize that before I could have made your change before because `TokenizerEnum['WORD']` and `/(\S+\s*)/g` were identical.